package.json

```diff
@@ -4,17 +4,20 @@
   "main": "test.js",
   "scripts": {
     "build": "npx babel src --out-dir lib",
-    "test": "cross-env NODE_ENV=test npm run unmigrate && cross-env NODE_ENV=test npm run migrate && cross-env NODE_ENV=test nyc --reporter=text --reporter=html  mocha --recursive --require @babel/polyfill --require @babel/register ./src/test/*.js --timeout 30000 --exit",
+    "test": "cross-env NODE_ENV=test npm run migrate  && cross-env NODE_ENV=test nyc --reporter=text --reporter=html  mocha --recursive --require @babel/polyfill --require @babel/register ./src/test/*.js --timeout 30000 --exit && cross-env NODE_ENV=test npm run unmigrate",
     "coverage": "nyc  --reporter=text-lcov --reporter=lcov npm run test | coveralls",
-    "start": "cross-env NODE_ENV=production npm run dbmigrate && node lib/index.js",
-    "dev": "nodemon --exec npx babel-node src/index.js",
+    "start": "cross-env NODE_ENV=production npm run migrate && cross-env NODE_ENV=production npm run dbseed  && node lib/index.js",
```

Comment:
You don't need to seed the data every time when app start it can cause app to crash because of the duplication emails on the user

we only need to run migration